src/components/src/ImageContainer/image.container.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright © 2019 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+
+import React from 'react';
+import { getConfig, IEpConfig } from '../utils/ConfigProvider';
+import imgPlaceholder from '../../../images/img_missing_horizontal@2x.png';
+
+let Config: IEpConfig | any = {};
+
+interface ImageContainerProps {
+  /** name of file */
+  fileName: string;
+  /** image URL */
+  imgUrl: string;
+  /** class name */
+  className?: string;
+  /** is a sku image */
+  isSkuImage?: boolean;
+  /** loading data */
+  onLoadData?: (...args: any[]) => any;
+}
+
+function ImageContainer(props: ImageContainerProps) {
+  const epConfig = getConfig();
+  Config = epConfig.config;
+  const {
+    fileName, imgUrl, className, isSkuImage, onLoadData,
+  } = props;
+
+  const fallbackTypes = Config.imageFileTypes.types;
+  const initType = imgUrl.replace(/.*(?=\.)/g, '');
+  const types = (fallbackTypes && fallbackTypes.length > 0) ? [
+    initType,
+    ...fallbackTypes.filter(fileType => fileType !== initType),
+  ] : [];
+  const imageSource = isSkuImage ? Config.skuImagesUrl.replace('%sku%', fileName) : Config.siteImagesUrl.replace('%fileName%', fileName);
+
+  const handleError = (e, defaultImgUrl) => {
+    const { src } = e.target;
+    if (Config.imageFileTypes.enable) {
```

Comment:
Let's also check if the top-level config is present too
```suggestion
    if (Config.imageFileTypes && Config.imageFileTypes.enable) {
```